lib/dockerregistry/inventory.go

```diff
@@ -1028,6 +1046,10 @@ func SplitByKnownRegistries(
 	r RegistryName,
 	rcs []RegistryContext) (RegistryName, ImageName, error) {
 
+	sort.Slice(rcs, func(i, j int) bool {
+		return len(rcs[i].Name) > len(rcs[j].Name)
```

Comment:
Deliberately sorting in the other order?

Reply:
Yes. I should have added a comment to explain (to echo my commit message). Will add a comment

Reply:
Done